### DISCUSSION TITLE
Ubuntu on a dell optiplex - power cycling issue

### DISCUSSION THREAD

**Discussion by dz4news** [blakeblackshear/frigate#7389] [Ask A Question]
Hopefully this is an easy question to answer... I am running frigate in docker on a dell optiplex.

When I have an unexpected power cycle, the system will restart and go to a welcome screen that requires my password. If the password is not entered in a certain amount of time the system will go into standby mode and I can not access it remotely. So if I am out of town, Frigate will be down until I get back and press the power button to wake the system and enter the password.
If I am able to remote into the system prior to it going into standby mode, I can enter the password and everything is fine.

Question... how do I disable the password and just let the system reboot or restart after a powercycle? 

I have a UPS that I have not setup yet... so I know what will most likely solve the issue also... 

TIA..

**Comment by moandcompany** [comment]
Check your Dell Optiplex system's BIOS configuration and disable any security settings that require an administrator or user password for booting an operating system.

**Comment by Cold-Lemonade** [comment]
@dz4news What version of Ubuntu are you running? Also, why are you running it with a desktop instead just using the command line?

**Reply by dz4news** [reply]
Ubuntu 22.04.3 LTS

**Reply by Cold-Lemonade** [reply]
I am not sure what is causing your unexpected power cycle. How often is this happening?

I used Ubuntu 22.04 LTS for nearly a year and never had an unexpected power cycle. But I was running it on a Minisforum GK41 as a headless server. What happens when your hardware goes into an unexpected power cycle? Is it like sleep mode on a laptop? If I had to guess, I think there might be a setting in the desktop that you're using that puts the computer to sleep after a certain amount of inactivity at the mouse or keyboard. Or perhaps it is a BIOS setting as @NickM-27 has suggested.

If you're using your Dell Optiplex as a server, then I would consider installing Ubuntu server and work exclusively from the command line.

**Reply by dz4news** [reply]
If and when we have a power outage (whole house), the system will come back on... but it goes to a screen that I have to put a password into. If I dont after a few minutes, it will go into standby mode and the power light will flash slowly until I press it. Once I press it and the screen comes back up for the password it will go back to working normally.
I am sure this is some setting I set when it was initially setup, I am just clueless on where to find it and disable it.
I have all the power set for " not" going into standby or sleep... so this is something that is being triggered at power up. I am leaning to a BIOS setting.
